modules/javafx.base/src/main/java/javafx/collections/FXCollections.java

```diff
@@ -347,9 +347,9 @@ public static <E> ObservableList<E> observableArrayList(Callback<E, Observable[]
      * @see #observableArrayList()
      */
     public static <E> ObservableList<E> observableArrayList(E... items) {
-        ObservableList<E> list = observableArrayList();
-        list.addAll(items);
-        return list;
+        ArrayList<E> backingList = new ArrayList<>(items.length);
+        backingList.addAll(Arrays.asList(items));
```

Comment:
Did you try using `List.of` instead?
```java
var backingList = new ArrayList<>(List.of(items));
```

Reply:
No, but `List.of` does not allow `null` values, therefore this would change the existing behaviour as `null` is allowed in an observable arraylist.